src/index.html

```diff
@@ -11,6 +11,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search__box">
+      <form
+        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+        method="post"
+        data-qa="big"
+      >
+        <img
+          class="icon1"
```

Comment:
```suggestion
          class="icon"
```
no need to add numbers for class
for small icon make classes icon and icon-small
same for other elements
![Screenshot from 2023-04-06 13-23-02](https://user-images.githubusercontent.com/33029864/230777519-f3cb8047-794b-4cef-a964-a3949c854f4f.png)